code/modules/clothing/gloves/tacklers.dm

```diff
@@ -98,3 +110,14 @@
 	base_knockdown = 1.75 SECONDS
 	min_distance = 2
 	skill_mod = -1
+
+/obj/item/clothing/gloves/tackler/guardian
+	name = "guardian gloves"
+	desc = "Fancy looking gloves passed down between generations of bodyguards. Landing a tackle on someone will temporarily grant you a defensive buff and bind your protectee to you, redirecting all attacks against them towards you."
+	icon_state = "tackledolphin"
+	inhand_icon_state = "tackledolphin"
+
+	tackle_range = 6
+	tackle_speed = 2
+	quip = "LOOK OUT, SIR!!"
```

Comment:
here's to hoping the captain is actually male 😉

Reply:
i was honestly considering checking the apparent gender of the person you clicked on to do "GET DOWN MR/MS/MX PRESIDENT" but then i realized that'd be a massive pain and people probably just click in the general direction of their target anyway, so stealing the name of the Warhammer rule that lets you make random grunts take a bullet for hero characters is good enough